osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -59,32 +60,33 @@ private void restartRequested()
         [BackgroundDependencyLoader]
         private void load()
         {
-            InternalChild = content = new Container
+            InternalChild = content = new LogoFacadeContainer
             {
                 Anchor = Anchor.Centre,
                 Origin = Anchor.Centre,
                 RelativeSizeAxes = Axes.Both,
-                Children = new Drawable[]
+            };
```

Comment:
can use `.WithChildren()`

Reply:
content.LogoFacade depends on the LogoTrackingContainer being constructed though, but I guess I can just assign the facade somehow afterwards

Reply:
![image](https://user-images.githubusercontent.com/191335/55539935-d366fc80-56fc-11e9-93ff-45cd110e0289.png)